setup.py

```diff
@@ -63,7 +63,8 @@
                 'progressbar2',
                 'boto3',
                 'requests',
-                'salem']
+                'salem',
+                'python-colorspace']
```

Comment:
Can you also remove it from setup.py?

Reply:
I'm just doing that now & about to squash it in (sorry, I didn't think anyone would be online at the present time).